packages/terra-section-header/src/SectionHeader.module.scss

```diff
@@ -161,7 +161,6 @@
     &.title-sticky {
       position: sticky;
       left: 12px;
-      width: auto;
```

Comment:
why are we removing this here ? are we expecting all the consumers of section-header to pass isTitleSticky as customprops ?

Reply:
**Bounded grids**

- When title is sticky and is a non-actionable header, we set to auto width: https://github.com/cerner/terra-core/pull/4043/files#diff-9bb4e4b8bfb491074cf911831bc7c9ec606b84abb69895cfe86dc2c758e8e270R166

- When title is sticky and actionable, we use boundedWidth.

**Non-bounded grids** 
We set width to 100%

isTitleSticky is passed only if grid is bounded